cli/mender-artifact/main.go

```diff
@@ -339,17 +333,30 @@ func getCliContext() *cli.App {
 			Usage: "Full path to the public verification key that is used by the client " +
 				"to verify the artifact.",
 		},
+		artifactName,
 		cli.StringFlag{
-			Name:  "name, n",
-			Usage: "New name of the artifact.",
+			Name:  "name",
```

Comment:
Suggestion: Is there no way to make them shadow each other, so that the before function is not needed?

Reply:
At least not in a way that would give it its own help text (otherwise we could simply have added `name` to the existing flag).

Reply:
I think you could use the `Destination` field, and make them share a variable, but then I'm not sure what `c.String("key")` will return. Anywho, it's nothing major.